usaspending_api/etl/management/commands/load_multiple_submissions.py

```diff
@@ -122,15 +141,18 @@ def get_incremental_submission_ids(self):
                 )
                 left outer join submission_attributes sa on
                     sa.submission_id = bs.submission_id and
-                    sa.certified_date is not distinct from bs.certified_date and
+                    sa.published_date::timestamp is not distinct from bs.published_date and
+                    sa.certified_date::timestamp is not distinct from bs.certified_date and
```

Comment:
Is this query used to identify "new" or "updated" submissions from broker to be loaded, or reloaded (delete old and reload new)?
Seeing this line `sa.certified_date::timestamp is not distinct from bs.certified_date`, it may be overzealous, or need more precision. 

Specifically, I think in broker, marking a submission certified _without_ first republishing it, requires that NO data in that submission changed since the initial publish. 

So, someone simply flagging a submission certified would trigger a complete delete and reload in USAspending. When all we really should do is update the certified date. (In fact, we probably don't even _need_ the certified date anymore. Not until "About the Data" needs to display it).

So maybe:
a) remove certified date completely if it's not critical to logic or display anymore (may be hard during this CARES transition back/fwd-compatibility phase)
b) Have a separate pre or post load check that just _updates_ certified dates if everything -- primarily publish date -- has stayed the same _except_ certified date.

Reply:
Agreed.  B) was actually my original intent, but got lost in the churn.